corehq/apps/app_manager/templates/app_manager/partials/nav_menu_media_single_type.html

```diff
@@ -58,19 +61,27 @@
                 {% endblocktrans %}
             </p>
             {% endif %}
-            <div>
+            <div data-bind="visible: isDefaultLanguage">
```

Comment:
Can this section not display when the app only has one language?

Reply:
https://github.com/dimagi/commcare-hq/pull/21443/commits/7f15e5b3f0cf80e2481d3c45199582eb2c8f8b9f